### ISSUE TITLE
How to force the extension to use the locally installed `php-cs-fixer` from the project (`executablePath`) instead of the default one ?

### ISSUE THREAD

**Issue by Alexandre-Fernandez** [junstyle/vscode-php-cs-fixer#205]
I installed `php-cs-fixer` locally and added some extensions. 
This extension reads the config file, sees the extensions and throws because it can't import them.
This happens because it's not using the `php-cs-fixer` from the project. When I change my VSCode setting and change the `executablePath` to an absolute path to the project `php-cs-fixer` `vendor` executable it works again.

**Comment by junstyle** [comment]
yes. Cantor read the path of the php-cs-fixer             that you install in the project. you must supply the path
